### ISSUE TITLE
icsnpp-s7comm package build fails in Zeek v6.0.0 release candidate

### ISSUE THREAD

**Issue by mmguero** [cisagov/icsnpp-s7comm#10]
```
Installing "https://github.com/cisagov/icsnpp-s7comm"
error: incomplete installation, the follow packages failed to be installed:
  https://github.com/cisagov/icsnpp-s7comm (main)
Failed installing "https://github.com/cisagov/icsnpp-s7comm": package build_command failed, see log in /opt/zeek-rc/var/lib/zkg/logs/icsnpp-s7comm-build.log
```

**Comment by mmguero** [comment]
```
> cat /opt/zeek-rc/var/lib/zkg/logs/icsnpp-s7comm-build.log
=== STDERR ===
Usage: zeek-config [OPTIONS]

Basic options:

  --build_type          Zeek build type as per cmake, lower case (e.g. 'relwithdebinfo')
  --prefix              Toplevel Zeek distribution installation directory
  --version             Zeek version number
  --zeek_dist           Toplevel directory of source tree the distribution built from
  --zeekpath            ZEEKPATH environment variable paths for this distribution

Specific directories in the Zeek distribution:

  --btest_tools_dir     Zeek-related BTest tooling
  --cmake_dir           Zeek's cmake modules
  --config_dir          Configuration files for cluster topology, zkg, etc
  --include_dir         C/C++ header folders for Zeek and related components, colon-separated
  --lib_dir             Toplevel folder for shared libraries, Python packages, etc
  --plugin_dir          Native-code Zeek plugins
  --python_dir          Python packages (Broker, ZeekControl, zkg, etc)
  --script_dir          Toplevel folder for Zeek scripts
  --site_dir            Site-specific Zeek scripts

Toplevel installation directories for third-party components:

  --binpac_root         BinPAC compiler
  --broker_root         Broker communication framework

Feature tests:

  --have-spicy-analyzers  Prints 'yes' if built-in Spicy analyzers are available; exit code reflects result

CMake Warning at /opt/zeek-rc/share/zeek/cmake/ZeekPlugin.cmake:141 (message):
  Package requires CMake 3.0 which is less than Zeek's requirement (3.15.0).
  This will likely cause build failures and should be fixed.
Call Stack (most recent call first):
  CMakeLists.txt:5 (include)


CMake Warning (dev) at /opt/zeek-rc/share/zeek/cmake/ZeekPlugin.cmake:120 (if):
  Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
  --help-policy CMP0057" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  /opt/zeek-rc/share/zeek/cmake/ZeekPluginDynamic.cmake:77 (zeek_next_pac_block)
  /opt/zeek-rc/share/zeek/cmake/ZeekPlugin.cmake:196 (zeek_add_dynamic_plugin)
  /opt/zeek-rc/share/zeek/cmake/ZeekPluginCommon.cmake:68 (zeek_add_plugin)
  CMakeLists.txt:12 (zeek_plugin_end)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at /opt/zeek-rc/share/zeek/cmake/ZeekPlugin.cmake:120 (if):
  if given arguments:

    "arg" "IN_LIST" "separators"

  Unknown arguments specified
Call Stack (most recent call first):
  /opt/zeek-rc/share/zeek/cmake/ZeekPluginDynamic.cmake:77 (zeek_next_pac_block)
  /opt/zeek-rc/share/zeek/cmake/ZeekPlugin.cmake:196 (zeek_add_dynamic_plugin)
  /opt/zeek-rc/share/zeek/cmake/ZeekPluginCommon.cmake:68 (zeek_add_plugin)
  CMakeLists.txt:12 (zeek_plugin_end)


=== STDOUT ===
Build Directory        : build
Zeek Source Directory   : 
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found OpenSSL: /usr/lib/x86_64-linux-gnu/libcrypto.so (found version "3.0.9")  
-- Found BinPAC: /opt/zeek-rc/bin/binpac  
-- Found BifCl at /opt/zeek-rc/bin/bifcl
-- Setting plugin CMAKE_BUILD_TYPE to Release
-- Configuring incomplete, errors occurred!
See also "/opt/zeek-rc/var/lib/zkg/clones/package/icsnpp-s7comm/build/CMakeFiles/CMakeOutput.log".
```

**Comment by mmguero** [comment]
Fixed by cisagov/icsnpp-s7comm@2a12f4f5a2e0446eec6f70e34a3a712050722559
